content/articles/how-is-dnsimple-pronounced.markdown

```diff
@@ -0,0 +1,27 @@
+---
+title: How to pronounce DNSimple
+excerpt: "It's not obvious how to pronounce DNSimple. Here's a guide."
+categories:
+- DNSimple
+---
+
+# The Name DNSimple
+
+Our goal has always been to simplify DNS, so DNSimple is a portmantou of the abbrevation for Domain Name System (DNS) and the English word Simple.
+
+But pronoucing it isn't always simple.
+
+Here's a basic guide:
+
+## Pronouncing DNSimple
+
+The easiest way to visualize it is D-N-Simple.
+
+Here at DNSimple we love sharing, that's why DNS and Simple share the "S".
```

Comment:
>Here at DNSimple we love sharing, that's why DNS and Simple share the "S"

This seems like something that makes more sense internally than externally. I think the initial explanation is enough without it.